resources/zib/zib-Procedure-event.xml

```diff
@@ -136,6 +140,7 @@
       <path value="Procedure.performed[x].end" />
       <short value="ProcedureEndDate" />
       <definition value="The end date (and if possible end time) of the procedure. A ‘vague’ date, such as only the year, is permitted. The element offers the option to indicate the end of the period of a series of related procedures. The end date element is only used for a procedures that takes some time and is then always applied. If the procedure still continues, the value is left empty. For instantaneous or very short lasting procedures the `performedDateTime` is used." />

```

Comment:
There is a small discrepancy in the `definition` of the element in the zib-Procedure-event/request profiles that is mapped on ProcedureEndDate, namely in the presence/absence of the sentence 'For instantaneous or very short lasting procedures the `performedDateTime` is used.' Should we remove this from the zib-Procedure-event as well, since the corresponding guidance is already present in the `.comment` below, or otherwise add it to zib-Procedure-request as well? My preference would be the former solution. 
(The origin of this sentence lies in the definition in the zib: 'For instantaneous or very short lasting procedures the element is omitted.')

Reply:
I've removed this last line, it was left over from the previous comment.